lib/config/fileconf_test.go

```diff
@@ -332,6 +334,34 @@ func TestAuthenticationSection(t *testing.T) {
 	}
 }
 
+func TestAuthenticationConfig_Parse_StaticToken(t *testing.T) {
+	roles := []string{"Auth", "Node", "Proxy"}
+	joinedRoles := strings.Join(roles[:], ",")
+	var token StaticToken = StaticToken(fmt.Sprintf("%s:token", joinedRoles))
+
+	provisionToken, err := token.Parse()
+	require.NoError(t, err)
+
+	require.Len(t, provisionToken.Roles, len(roles))
+	for i := range roles {
+		require.Equal(t, string(provisionToken.Roles[i]), roles[i])
+	}
+	require.Equal(t, provisionToken.Token, "token")
```

Comment:
Simplify?

```suggestion
	want := &types.ProvisionTokenV1{
		Roles: []types.SystemRole{
			types.RoleAuth, types.RoleNode, types.RoleProxy,
		},
		Expires: provisionToken.Expires, // or replace with the value you want
		Token: "token",
	}
	require.Equal(t, want, provisionToken)
```

it's easier to read/reason about and has a lot less logic that could go wrong.